tests/components/homekit_controller/specific_devices/test_lg_tv.py

```diff
@@ -62,3 +62,7 @@ async def test_lg_tv(hass):
     assert device.model == "OLED55B9PUA"
     assert device.sw_version == "04.71.04"
     assert device.via_device_id is None
+
+    # A TV doesn't have any triggers
+    triggers = await async_get_device_automations(hass, "trigger", device.id)
+    assert triggers == []
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6158285)